docs/cli/ilab-local-config.md

```diff
@@ -0,0 +1,184 @@
+| Publish | Model checkpoints to be published | n/a |
+
+
+Based on this, it's evident that we need a few different subdirectories:
+
+| path | description |
+|------|-------------|
+| `base_models/` | Stores the base models that we download |
+| `checkpoints/` | Houses the checkpoints we generate during training. Can potentially split these out further into `checkpoints/phase00`, `checkpoints/phase05`, and `checkpoints/phase10` |
+| `datasets` | Would contain the generated datasets. Each item here would be a directory consisting of: `train.jsonl` and `test.jsonl` |
+| `logs` | Would contain logs from various `ilab` runs |
+| `eval_data` | Contains scores from various evaluation runs for a given model ref |
+
+
+## How ilab would manage this data
+
+There should be a ***simple and intuitive*** way to view & manage all of this data.
+
+The basic idea is that each piece of information can be discovered through a `list` command in some form. Each item in the listing will have the corresponding associated information, along with a unique reference that the user can use when interacting with subsequent commands.
+
+
+### Manging model data
+
+
+New model checkpoints would be generated during training and
+saved to the `checkpoints/` directory along with a unique ref. 
+When base models are downloaded, they would be saved in the `base_models` directory with their basename being the unique ref.
+
+
+An `ilab model list` command would be added which allows the user to list out existing models. This would allow users to view all models, base models, trained checkpoints, etc. Additionally, the user could further filter out trained checkpoints based on which training phase they were generated in, i.e. `phase00`, `phase05`, and `phase10`.
+
+We can iterate on the behavior and optimize for UX, but the
+idea is you should be able to run either: `ilab model list` or
+`ilab list models` which produces a full list of all models on the local system.
+
+The output of `ilab model list` would look like the following:
+
+```
+MODEL ID    BASE MODEL                    SIZE    DATA TYPE    DATASET TRAINED WITH
+abcde       ibm-granite/granite-7b-base   28Gi    FP32         phase00-dataset-id
+cbdea       abcde                         28Gi    FP32         phase05-dataset-id
+```
+
+
+
+Finally, we would add a new `ilab model delete <ref>`  command 
+which would enable us to pass a reference to the model we want
+to delete.
+
+
+### Managing datasets
+
+New datasets would be generated through `ilab generate` (or whatever the current equivalent is) and the corresponding dataset would be saved under the `datasets/` directory. Each model would also have a unique ref associated with it.
+
+Following this, we could list out the datasets with `ilab dataset list` and further filter them out with either `knowledge` and `skills`, or `phase00`, `phase05`, and `phase10` depending on which SDG scheme is being used.
```

Comment:
Is `dataset` a scoped command group? We already have `data` and `model` - do we need another group?

Reply:
I'm not familiar with data but it sounds like that's where it would go under.

Reply:
If that's the case please update the proposal to reflect that